api/v1alpha1/linodeobjectstoragebucket_types.go

```diff
@@ -43,6 +43,11 @@ type LinodeObjectStorageBucketSpec struct {
 	// +optional
 	// +kubebuilder:default=0
 	KeyGeneration *int `json:"keyGeneration,omitempty"`
+
+	// secretType sets the type for the access-keys secret that will be generated by the controller.
```

Comment:
nit: consistent with other field comments

```suggestion
	// SecretType sets the type for the access-keys secret that will be generated by the controller.
```